packages/atlas/src/views/studio/VideoWorkspace/VideoWorkspace.tsx

```diff
@@ -11,6 +12,8 @@ import { VideoWorkspaceForm } from './VideoWorkspaceForm'
 import { VideoWorkspaceTabsBar } from './VideoWorkspaceTabsBar'
 
 export const VideoWorkspace: React.FC = React.memo(() => {
+  const headTags = useHeadTags('Edit video')
```

Comment:
Shouldn't we distinguish between adding end editing?

Reply:
Wanted to keep it simple, but updated now, also for channel edit